### PR TITLE
start of node rewrite

### PR DESCRIPTION
**Pull request by faiq**

### CODE REVIEW

**Review by rsakallah — COMMENTED** [review]
Where is the XML coming from?

File: house/house.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const fs = require('fs')
+const xml2js = require('xml2js')
+
+BillFetcher.prototype.FetchBills = function (date, cb) {
```
Comment (rsakallah) [inline house/house.js:4]: What is cb?

Comment (faiq) [reply house/house.js:4]: cb is short for callback, which is a function that will execute when an asynchronous request is done... 


https://en.wikipedia.org/wiki/Callback_(computer_programming)

**Comment by faiq** [comment]
The XML is coming from a file, right now... we'll be using files to do a lot of the testing, rather than making the request to the congress API every time

File: house/house.js (hunk 1 above)
Comment (rsakallah) [reply house/house.js:4]: dab

https://www.youtube.com/watch?v=T9MZJoW0v6A

**Pull request merged**